static-site/components/list-resources/modal_draw.js

```diff
@@ -92,29 +92,25 @@ export default function modal_draw(ref, resource, lightGrey) {
     heights.marginBelowAxis;
   let radius = 12;
   const padding = 1;
-  let nextRadius = radius;
   let iterCount = 0;
   let beeswarmData;
   let beeswarmHeight = 0;
   let spareHeight = availBeeswarmHeight - beeswarmHeight - radius;
   const maxIter = 5;
   const radiusJump = 2;
   while (iterCount++ < maxIter && spareHeight > 50) {
-    const nextBeeswarmData = dodge(flatData, {
-      radius: nextRadius * 2 + padding,
+    beeswarmData = dodge(flatData, {
+      radius: radius * 2 + padding,
       x: (d) => x(d["date"]),
     });
-    const nextBeeswarmHeight = d3.max(nextBeeswarmData.map((d) => d.y));
+    beeswarmHeight = d3.max(beeswarmData.map((d) => d.y));
     const nextSpareHeight =
-      availBeeswarmHeight - nextBeeswarmHeight - nextRadius;
+      availBeeswarmHeight - beeswarmHeight - radius;
     if (nextSpareHeight <= spareHeight && nextSpareHeight > 0) {
-      beeswarmData = nextBeeswarmData;
-      beeswarmHeight = nextBeeswarmHeight;
       spareHeight = nextSpareHeight;
-      radius = nextRadius;
-      nextRadius += radiusJump;
+      radius += radiusJump;
     } else {
-      nextRadius -= radiusJump;
+      radius -= radiusJump;
```

Comment:
This fixes the bug, but the radius seems off:

<img width="707" height="224" alt="Image" src="https://github.com/user-attachments/assets/2ef1149a-a73f-4968-8d45-d312c1582777" />